### ISSUE TITLE
Components sandbox / playground

### ISSUE THREAD

**Issue by anbalase** [UiPath/angular-components#68]
Based on @vladjerca's suggest sandbox setup 496fde74356d52b4cc62cdff3d7885eaa0ccb3bd expand for all components

**Comment by vladjerca** [comment]
@anbalase @mhagape I think we could use [storybook](https://storybook.js.org/docs/guides/guide-angular/) to document usage and display the rendered results.

All our libs are going this direction and it would be great if we could add this to the `gh_pages`. Maybe drop `compodoc`?

**Comment by sergiubologa** [comment]
> @anbalase @mhagape I think we could use [storybook](https://storybook.js.org/docs/guides/guide-angular/) to document usage and display the rendered results.
> 
> All our libs are going this direction and it would be great if we could add this to the `gh_pages`. Maybe drop `compodoc`?

We tried here in Cluj to make Storybook work with the lib and it didn't work quite well. We couldn't make it work properly. I didn't invest too much time, maybe you'll have more luck.

**Comment by vladjerca** [comment]
@sergiubologa Was it because of the pdf-js web worker or what exactly was the issue?

**Comment by sergiubologa** [comment]
> @sergiubologa Was it because of the pdf-js web worker or what exactly was the issue?

Not really. Mainly it was because the DI doesn't seem to work with Storybook and I couldn't find a quick solution for it. Maybe the upgrade to Angular 9 will fix the issue.
